src/proj.h

```diff
@@ -182,27 +262,93 @@ PJ_OBSERVATION pj_apply (PJ *P, enum pj_direction direction, PJ_OBSERVATION obs)
 /* Measure internal consistency - in forward or inverse direction */
 double pj_roundtrip(PJ *P, enum pj_direction direction, int n, PJ_OBSERVATION obs);
 
-
 int pj_show_triplet (FILE *stream, const char *banner, PJ_TRIPLET point);
 
-/* Constructor for the OBSERVATION object */
+PJ *pj_pj (PJ_CONTEXT *ctx, char *definition);
```

Comment:
1. const char* would be better
2. pj_pj() isn't really explicit. pj_create_from_def() maybe ?
3. This is a more general question, but how should we document functions and structures. Informally here as comments, or with Doxygen formalism in the .c file (or in .h) ? Update: I see we have https://github.com/OSGeo/proj.4/tree/master/docs/source/development now, so that would be the place for that